arelle/Validate.py

```diff
@@ -298,6 +298,7 @@ def _validateTestcaseVariation(self, testcase, modelTestcaseVariation):
             self._testcaseValidateInputDTS(testcase, modelTestcaseVariation, errorCaptureLevel, parameters, inputDTSes, baseForElement, resultIsXbrlInstance)
         # update ui thread via modelManager (running in background here)
         self.modelXbrl.modelManager.viewModelObject(self.modelXbrl, modelTestcaseVariation.objectId())
+        self.modelXbrl.modelManager.cntlr._clearPluginData()
```

Comment:
Why does this need to be called here?

Reply:
That makes sense. It's unfortunate that the test case variation discovery is handled within the validation function.